server/utils/firebaseAdmin.js

```diff
@@ -58,19 +59,23 @@ const maybeAddKeyTimestampPair = (key, timestamp) => {
 	return null;
 };
 
-export const getBranchDoc = async (pubId, branchId, historyKey, updateOutdatedCheckpoint) => {
+export const getBranchDoc = async (pubId, branchId, historyKey, createMissingCheckpoints) => {
 	const branchRef = getBranchRef(pubId, branchId);
 
 	const [
-		{ doc, key: currentKey, timestamp: currentTimestamp, checkpointMap },
+		{ doc, docIsFromCheckpoint, key: currentKey, timestamp: currentTimestamp, checkpointMap },
 		{ timestamp: firstTimestamp, key: firstKey },
 		{ timestamp: latestTimestamp, key: latestKey },
 	] = await Promise.all([
-		getFirebaseDoc(branchRef, editorSchema, historyKey, updateOutdatedCheckpoint),
+		getFirebaseDoc(branchRef, editorSchema, historyKey, createMissingCheckpoints),
```

Comment:
Are we using this fourth argument any more? Looks like it was removed.

Reply:
Right you are, good catch!